.github/workflows/linux_build_tests.yaml

```diff
@@ -26,7 +27,14 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        package: [lz4, mpich, tut, py-setuptools, openjpeg, r-rcpp]
+        package:
+        - lz4  # MakefilePackage
+        - mpich  # AutotoolsPackage
+        - tut  # WafPackage
+        - py-setuptools  # PythonPackage
+        - openjpeg  # CMakePackage
+        - r-rcpp  # RPackage
+        - ruby-rake  # RubyPackage
```

Comment:
`ruby-rake` only takes a couple of seconds to build since I added `ruby` as an external package.